locale/en.po

```diff
@@ -1032,75 +1032,75 @@ msgstr "Viewer"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:162
 msgid "ALLOW_SPECIES_TO_NOT_MUTATE"
-msgstr "Allow species to not mutate"
+msgstr "Allow species to not mutate (if no good mutation is found)"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:169
 msgid "ALLOW_SPECIES_TO_NOT_MIGRATE"
-msgstr "Allow species to not migrate"
+msgstr "Allow species to not migrate (if no good migration is found)"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:182
 msgid "BIODIVERSITY_ATTEMPT_FILL_CHANCE"
-msgstr "Biodiversity attempt fill chance"
+msgstr "Chance in each patch with few species (low biodiversity) to create a new species"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:202
 msgid "BIODIVERSITY_FROM_NEIGHBOUR_PATCH_CHANCE"
-msgstr "Biodiversity from neighbour patch chance"
+msgstr "Chance to create a new species to increase biodiversity form a nearby patch"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:216
 msgid "BIODIVERSITY_NEARBY_PATCH_IS_FREE_POPULATION"
-msgstr "Biodiversity nearby patch is free population"
+msgstr "Biodiversity increasing species from nearby patch is given free population"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:223
 msgid "BIODIVERSITY_SPLIT_IS_MUTATED"
-msgstr "Biodiversity split is mutated"
+msgstr "Biodiversity increasing species is mutated on creation"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:236
 msgid "LOW_BIODIVERSITY_LIMIT"
-msgstr "Low biodiversity limit"
+msgstr "Consider patches with up to this many species having low biodiversity"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:254
 msgid "MAXIMUM_SPECIES_IN_PATCH"
-msgstr "Maximum species in patch"
+msgstr "Maximum species in a patch before forced extinctions"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:272
 msgid "MOVE_ATTEMPTS_PER_SPECIES"
 msgstr "Move attempts per species"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:290
 msgid "MUTATIONS_PER_SPECIES"
-msgstr "Mutations per species"
+msgstr "Mutation attempts per species"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:308
 msgid "NEW_BIODIVERSITY_INCREASING_SPECIES_POPULATION"
-msgstr "New biodiversity increasing species population"
+msgstr "Initial population for biodiversity increasing species"
 
 #: ../src/auto-evo/AutoEvoExploringTool.tscn:322
 msgid "PROTECT_MIGRATIONS_FROM_SPECIES_CAP"
-msgstr "Protect migrations from species cap"
+msgstr "Protect just migrated populations from species cap"
```

Comment:
Is "species cap" "force extinction"? If so, I think directly say "from force extinction" is better.

Reply:
I don't want to constraint people wanting to change the auto-evo too much by being overtly specific here.
I don't particularly like forced extinctions as a concept so I'd like us to move away from that in terms of the species cap, so I'd prefer to leave this as a more general way to say this.

Reply:
I'll merge this to get these changes on weblate for translators. If there's some better wording (that isn't too much longer as we need to be careful with these as these don't have line wrapping enabled) a suggestion can be made on weblate.